### DISCUSSION TITLE
How to extract text from PDF url

### DISCUSSION THREAD

**Discussion by aleksandar-devedzic** [py-pdf/pypdf#1646] [Q&A] [answered]
Is there a way to parse PDF directly from URL?
Can you post example code?
Here a couple of PDFs:

https://www.niederbipp.ch/_docn/3783791/Protokoll_der_Versammlung_vom_13.6.2022_geschwarzt.pdf
https://www.niederglatt-zh.ch/_docn/3767428/2022.06.08_Publikation_GV_Beschlusse_sw.pdf

**Comment by pubpub-zz** [comment, marked as answer]
pypdf has no capabilities to load from URL. However `get_pdf_from_url()` can be used as an example to get the data.
source code in https://github.com/py-pdf/pypdf/blob/main/tests/__init__.py
many examples are available in the different test files in https://github.com/py-pdf/pypdf/blob/main/tests
